### ISSUE TITLE
Feature: About Page: Create Blockquote Component with changing Quotes [Stretch Goal]

### ISSUE THREAD

**Issue by sm3dev** [sm3dev/michaelpwright-portfolio#33]
**Issue #32 has to be completed first.**
Create component for blockquotes used on the About Page view. The quotes and citations should change every 12 or 24 hours.

### Actions
- [x] Creat component file and add HTML for blockquote and cite elements
- [x] Add `aboutQuotes` table to database and give the following properties: `id`, `quoteText`, `author` 
- [x] Write HTML for Blockquote element including a place for each property value of the `aboutQuotes` object
- [x] Create a function that fetches the `aboutQuotes` objects from the database and puts them into a new array
- [ ] import Helper function from Issue #32 to loop through the array and select one object every 24 hours 

### Considerations:
<cite> element should be all caps with text small than blockquote quote
